extension/test/bindings/contextual_bindings.cpp

```diff
@@ -33,7 +33,7 @@ int main() {
   // clang-format off
   auto injector = di::make_injector<di::extension::contextual_bindings>(
       di::bind<>().to(123.f)
-    , di::bind<int>().to([](const auto& injector) {
+    , di::bind<int>().to([](const auto& injector) -> int {
```

Comment:
This one is really weird. I understand that it might be an issue to deduct type from injector. But this one is trivial 😞